tufrepo/cli.py

```diff
@@ -288,18 +278,42 @@ def set_expiry(ctx: Context, expiry: Tuple[int, str]):
 
 @edit.command()
 @click.pass_context
-@click.argument("delegate")
-def add_key(ctx: Context, delegate: str):
-    """Add new signing key for delegated role DELEGATE
+@click.option("--delegate")
+def add_key(ctx: Context, delegate: Optional[str] = None):
```

Comment:
Hmm, does adding `--delegate` add value? The commands are already so long I'd rather not make them longer if it's not required... 

Are there issues with just making the argument optional:
```python
@click.argument("delegate", required=False)
def add_key(ctx: Context, delegate: Optional[str]):
```

(btw, the default values in function signatures should not be needed: the caller is Click and it will always provide a value)